app/src/main/java/org/fossasia/phimpme/gallery/activities/ExcludedAlbumsActivity.java

```diff
@@ -49,6 +56,35 @@ private void checkNothing() {
         a.setVisibility(excludedFolders.size() == 0 ? View.VISIBLE : View.GONE);
     }
 
+    public void restoreAll(int itemcount){
+        if (excludedFolders.size()!=0){
+            for (int i=0;i<itemcount;i++){
+                h.clearAlbumExclude(excludedFolders.remove(0).getAbsolutePath());
+            }
+            SnackBarHandler.show(findViewById(R.id.rl_ea), "All folders are restored", Snackbar.LENGTH_SHORT).show();
+            new Handler().postDelayed(new Runnable() {
```

Comment:
What is the use of the postDelayed handler and finish() ?

Reply:
as soon as all the folders are restored, its better not to show the blank screen

Reply:
ok